internal/api/token_test.go

```diff
@@ -213,13 +213,13 @@ func (ts *TokenTestSuite) TestSingleSessionPerUserNoTags() {
 	assert.True(ts.T(), ts.API.config.Sessions.SinglePerUser)
 
 	var firstResult struct {
-		Error            string `json:"error"`
-		ErrorDescription string `json:"error_description"`
+		ErrorCode string `json:"error_code"`
+		Message   string `json:"msg"`
 	}
 
 	assert.NoError(ts.T(), json.NewDecoder(w.Result().Body).Decode(&firstResult))
-	assert.Equal(ts.T(), "invalid_grant", firstResult.Error)
-	assert.Equal(ts.T(), "Invalid Refresh Token: Session Expired (Revoked by Newer Login)", firstResult.ErrorDescription)
+	assert.Equal(ts.T(), ErrorCodeSessionExpired, firstResult.ErrorCode)
+	assert.Equal(ts.T(), "Invalid Refresh Token: Session Expired (Revoked by Newer Login)", firstResult.Message)
```

Comment:
/nit Perhaps considered already - it looks like we distinguish between (Inactivity) and (Revoked by Newer Login) so might be worth separating the error codes so in future there's no dependency on the error message.

Aware that it is the current behaviour now though (to encode the cause in the error message) so it's probably fine to leave as is too.

Reply:
yeah was thinking that this doesn't have to be split up because the grouping is identical here - the session is expired